packages/v4/src/content/community/extensions-data.js

```diff
@@ -0,0 +1,58 @@
+export const communityExtensions = [
+  {
+    "component": "Catalog View",
+    "description": "Implements a catalog view using PatternFly components",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/catalog-view-item-header"
+      },
+      {
+        "name": "Code",
+        "href": "https://www.npmjs.com/package/@patternfly/react-catalog-view-extension"
+      }
+    ]
+  },
+  {
+    "component": "Console",
+    "description": "React-console extension",
```

Comment:
```suggestion
    "description": "Implements a React-console extension.",
```

Let's write these descriptions in a parallel way ("Implements + information.") so that they're easier for Flyers to scan!